ddtrace/tracer.py

```diff
@@ -73,15 +77,20 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None):
         if sampler is not None:
             self.sampler = sampler
 
-    def trace(self, name, service=None, resource=None, span_type=None):
-        """Return a span that will trace an operation called `name`.
+    def trace(self, name, service=None, resource=None, span_type=None, ctx=None, span_parent=None):
```

Comment:
actually I think that a good idea is always infer the context from the span if it's present. Something like: https://github.com/opentracing/opentracing-python/blob/master/opentracing/tracer.py#L72-L74